### PR TITLE
Update skype-room-systems-v2.md

### PR DESCRIPTION
**Pull request by MarcoCarampanta**
Please engage me if any additional clarification is required.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/room-systems/skype-room-systems-v2.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Microsoft has and dedicated SKU for licensing meetings and calling on a per-devi
 
 
 > [!NOTE]
-> If you are currently using E1, E3, E4, E5 SKUs with Skype for Business Standalone Plan 1 or Skype for Business Plan 2 or Skype for Business Plan 2 with Audio conferencing or with Office365 Phone System and a Calling Plan, these will continue to work. However, you should consider moving to a simpler licensing model in the table above after current licenses expire.  
+> If you are currently using E1, E3, E4, E5 SKUs with Skype for Business Plan 2 with Audio conferencing or with Office365 Phone System and a Calling Plan, these will continue to work. However, you should consider moving to a simpler licensing model in the table above after current licenses expire.
```
Comment (nenonix) [inline Teams/room-systems/skype-room-systems-v2.md]:
```suggestion
> If you are currently using E1, E3, E4, E5 SKUs with Skype for Business Plan 2 with Audio Conferencing or with Office 365 Phone System and a Calling Plan, these will continue to work. However, you should consider moving to a simpler licensing model in the table above after current licenses expire.  
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
@MarcoCarampanta - can you commit the edits made by nenonix?  Thank you.

**Pull request merged**